charts/unspec-service/values.preview.template.yaml

```diff
@@ -1,12 +1,43 @@
 tags:
   unspec-ccd-stack: true
 
+postgresql:
+  enabled: true
+  image:
+    tag: '11.3.0'
+  resources:
+    requests:
+      cpu: 25m
+  initdbScripts:
+    init.sql: |-
+      CREATE DATABASE cmc WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+      CREATE DATABASE "data-store" WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+      CREATE DATABASE "definition-store" WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+      CREATE DATABASE "user-profile" WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+  postgresqlUsername: hmcts
+  postgresqlPassword: hmcts
+  persistence:
+    enabled: false
+
 java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  keyVaults:
+    unspec:
+      resourceGroup: unspec
+      secrets:
+        - unspec-idam-client-secret
+        - microservicekey-unspec-service
+        - system-update-user-username
+        - system-update-user-password
+        - AppInsightsInstrumentationKey
```

Comment:
this shouldn't be required as it's in base `values.yaml`

Reply:
just to override the vaults config in values.yaml as values.yaml have more specific for env config

Reply:
as discussed, this is required, resolving